samples/react-native/src/Screens/ErrorsScreen.tsx

```diff
@@ -248,6 +248,7 @@ const ErrorsScreen = (_props: Props) => {
         ) : null}
         <View style={styles.mainViewBottomWhiteSpace} />
       </ScrollView>
+      <FeedbackButton navigation={_props.navigation} />
```

Comment:
Since this is a sample app, people may use it as a reference to implement Sentry features.
At the moment, the feedback button will always cover the last items, we should make it so that when users scrolls to be bottoms that the button will not cover the last views from the screen.

Reply:
Good point @lucas-zimerman 👍 
I've added some extra spacing at the bottom with fbc8f7712ba4928b4f45442d9b6d5c3039131d14

<img width="727" alt="Screenshot 2024-12-18 at 5 10 08 PM" src="https://github.com/user-attachments/assets/cad68b07-2238-4376-a224-7d25537e308a" />